core/src/main/java/hudson/tasks/UnstableAwareCommandInterpreter.java

```diff
@@ -0,0 +1,39 @@
+package hudson.tasks;
+
+import hudson.Util;
+import hudson.util.FormValidation;
+import org.kohsuke.accmod.Restricted;
+import org.kohsuke.accmod.restrictions.DoNotUse;
+import org.kohsuke.stapler.QueryParameter;
+
+public abstract class UnstableAwareCommandInterpreter extends CommandInterpreter {
+    public UnstableAwareCommandInterpreter(String command) {
+        super(command);
+    }
+
+    static String invalidExitCodeZero() { return null; };
```

Comment:
Do they need to be static?

Reply:
The problem is that the caller is:
```
    public static final class DescriptorImpl extends BuildStepDescriptor<Builder> {
...
        public FormValidation doCheckUnstableReturn(@QueryParameter String value) {
            return helpCheckUnstableReturn(value);
```

I'm not sure how to get past this. I guess the way to do it is to pass an object to `helpCheckUnstableReturn`